tests/nightly/broken_link_checker_test/JenkinsfileForBLC

```diff
@@ -53,14 +53,16 @@ try {
     parallel 'BrokenLinkChecker: CPU': {
       node('mxnetlinux-cpu') {
         ws('workspace/brokenLinkChecker') {
-		  try {
-          	init_git()
-		  	sh 'aws s3 cp s3://mxnet-ci-prod-slave-data/url_list.txt  ./tests/nightly/broken_link_checker_test/url_list.txt'
-		  	docker_run('ubuntu_blc', 'broken_link_checker', false)
-		  } finally {
-		  	sh "echo Storing the new url_list.txt to S3 bucket" 
-		   	sh 'aws s3 cp ./tests/nightly/broken_link_checker_test/url_list.txt s3://mxnet-ci-prod-slave-data/url_list.txt'
-		  }
+          timeout(time: 60, unit: 'MINUTES') {
```

Comment:
How long does the link checker typically take?

Reply:
30 minutes

Reply:
Data such as mean and variance would be helpful in informing this setting.

Reply:
The data is available at http://jenkins.mxnet-ci.amazon-ml.com/job/Broken_Link_Checker_Pipeline/

Reply:
40m seems enough

Reply:
Done